.github/workflows/build-pr-artifacts.yml

```
@@ -70,3 +70,10 @@
       push_target: production
     secrets:
       DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+
+  run-ingestion-service-test:
+    name: Run Ingestion Service Test
+    needs: [build-transformer-image]
+    uses: ./.github/workflows/ingestion-service-test.yml
+    with:
+      build_tag: rudderstack/develop-rudder-transformer:${{ needs.generate-tag-names.outputs.tag_name }}
```

actionlint (1.7.4)

property "generate-tag-names" is not defined in object type {build-transformer-image: {outputs: {}; result: string}}

(expression)